tendrl/commons/objects/node/atoms/stop_integration_services/__init__.py

```diff
@@ -0,0 +1,58 @@
+from tendrl.commons import objects
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons.utils import log_utils as logger
+
+
+class StopIntegrationServices(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(StopIntegrationServices, self).__init__(*args, **kwargs)
+
+    def run(self):
+        services = ["tendrl-gluster-integration"]
```

Comment:
Write a utility function which takes list of services and stops+disables the services. The same utility could be called from both the atoms, `StopMonitoringServices` and `StopIntegrationServices`.

Reply:
Sure, Will do it